src/EnergyPlus/DemandManager.cc

```diff
@@ -207,7 +207,7 @@ void SimulateDemandManagerList(EnergyPlusData &state,
     //       RE-ENGINEERED  na
 
     // Using/Aliasing
-    auto &TimeStepSys = state.dataHVACGlobal->TimeStepSys;
+    Real64 TimeStepSysSec = state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
```

Comment:
The one other change I made was to create a shortcut for this common idiom, either instead of or in addition to shortcutting `TimeStepSys` itself.

Reply:
@amirroth I like the wider use of TimeStepSysSec but in a bunch of places it is surrounded by parenthesizes that are now unnecessary and reduce readability.

Reply:
How about adding `TimeStepSysSec` to `state` and compute it only once at the same time `TimeStepSys` is set (which could be more than one place due to downstepping).

Reply:
> @amirroth I like the wider use of TimeStepSysSec but in a bunch of places it is surrounded by parenthesizes that are now unnecessary and reduce readability.

You've uncovered my secret which is that I did this largely using search-and-replace. I can search for this pattern and replace it also.

Reply:
> How about adding `TimeStepSysSec` to `state` and compute it only once at the same time `TimeStepSys` is set (which could be more than one place due to downstepping).

I thought of doing this, but I was afraid I wouldn't find all the places in which `TimeStepSys` is modified (It had already bitten me once). I can give it a shot.

Reply:
> > How about adding `TimeStepSysSec` to `state` and compute it only once at the same time `TimeStepSys` is set (which could be more than one place due to downstepping).
> 
> I thought of doing this, but I was afraid I wouldn't find all the places in which `TimeStepSys` is modified (It had already bitten me once). I can give it a shot.

Is `HVACManager.cc` the only place where this happens?

Reply:
> Is `HVACManager.cc` the only place where this happens?

Yes.